app/packs/src/apps/mydb/elements/details/reactions/analysesTab/ReactionDetailsContainers.js

```diff
@@ -56,10 +58,29 @@ export default class ReactionDetailsContainers extends Component {
     this.handleOnClickRemove = this.handleOnClickRemove.bind(this);
     this.handleAccordionOpen = this.handleAccordionOpen.bind(this);
     this.handleSpChange = this.handleSpChange.bind(this);
+    this.onUIStoreChange = this.onUIStoreChange.bind(this);
   }
 
   componentDidMount() {
+    const { activeContainer } = this.state;
+    UIStore.listen(this.onUIStoreChange);
     TextTemplateActions.fetchTextTemplates('reaction');
+    if (this.containerRefs[activeContainer]) {
+      this.containerRefs[activeContainer].scrollIntoView({ behavior: 'smooth', block: 'start' });
```

Comment:
Fair point. Generally, I'd also be irritated by auto-scrolling. This situation is an exceptions, the auto-scrolling is helpful because it saves you from having to scroll down a potentially long list of analyses to find the one you're jumping to (from the `Variations` tab). You can try it out by adding a bunch of analyses and then jumping to one that's off screen.

Reply:
I just checked this with a long list of analyses, and I agree for the case that you jump from Variations using the link to the Analyses.
But the auto-scrolling is also applied, when I simply manually went to the `Analyses` tab. Is this intended?

Reply:
The scrolling on manual selection of the `Analyses` tab is an unintended side-effect. I just opened the tab manually and you're right, the scrolling feels weird. I didn't find an easy fix. However, I changed the scrolling behavior to `instant`, and now you won't perceive any scrolling anymore / it doesn't feel like the UI does sth "against your will". With the `instant` behavior the tab is simply always opened with the currently active analysis. I think this is actually nice, since I believe it's reasonable to assume that users care most about the currently active analysis. See c848f0e58a76dec296a2f5804e0ce6595ce396f0.

Reply:
Agree, I checked this locally, the instant scrolling feels much better. Thanks for your engagement. :)